packages/editor/src/components/rich-text/input-event.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export class RichTextInputEvent extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.onInput = this.onInput.bind( this );
+	}
+
+	onInput( event ) {
+		if ( event.inputType === this.props.inputType ) {
+			this.props.onInput();
+		}
+	}
+
+	componentWillMount() {
```

Comment:
`componentDidMount` would be more appropriate for the side-effecting event listener, also considering that `componentWillMount` and _not_ `componentWillUnmount` are called in a `renderToString` context (not that it would happen here, but it's [easy to overlook](https://github.com/WordPress/gutenberg/pull/11819)).

https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9

Reply:
Thanks for the tip!